### PR TITLE
Add browserless image, use to ping slack with report screenshot

### PR DESCRIPTION
**Pull request by colophonemes**

### CODE REVIEW

**Comment by jpaddison3** [comment]
Just checking, did you forget to add a file to the commit?

**Comment by colophonemes** [comment]
Turns out that transfer.sh has intermittent DNS problems, so I've updated to use Backblaze B2. Credentials are in the tech vault for testing.

File: tasks/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM alpine
 
-RUN apk --update --no-cache add jq curl postgresql-client
+RUN apk --update --no-cache add jq curl postgresql-client py2-pip
```
Comment (jpaddison3) [inline tasks/Dockerfile:3]: Why py2?

**Review by jpaddison3 — APPROVED** [review]
Very nice
ASTLAC

File: tasks/Dockerfile (hunk 1 above)
Comment (colophonemes) [reply tasks/Dockerfile:3]: Needed to pull `b2` on the following line. Alpine Linux is _super_ minimal

**Pull request merged**